### PR TITLE
feat: Add Roshan-Sharma profile 

### PR DESCRIPTION
**Pull request by imrosun**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/1525"></a>

### CODE REVIEW

**Review by Panquesito7 — APPROVED** [review]
Thanks for contributing! 🚀


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good&nbsp; | &nbsp;**Tags:** Maintainable

File: public/data/Roshan-Sharma.json [hunk 1]
```diff
@@ -0,0 +1,32 @@
+{
+  "name": "Roshan Sharma",
+  "type": "personal",
+  "bio": "I have completed my graduation in 2020. I have done MCA with a CGPA of 7.57/10. I am currently looking for a Software Developer role. I am open to collaborate with any open source project to increase my skills.",
+  "avatar": "https://github.com/RonenTGreat.png",
```
Comment (kumarsonsoff3) [inline public/data/Roshan-Sharma.json:5]: You might have entered the wrong URL here! 

```suggestion
  "avatar": "https://github.com/imrosun.png",
```

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

**Review by kumarsonsoff3 — CHANGES_REQUESTED** [review]
Please update your **avatar** link, it's from someone else profile! 


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

**Review by imrosun — COMMENTED** [review]
Added avatar png link

**Review by kumarsonsoff3 — APPROVED** [review]
__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good&nbsp; | &nbsp;**Tags:** Elegant, Maintainable

**Pull request merged**